src/inc/quic_crypt.h

```diff
@@ -231,6 +231,16 @@ QuicPacketKeyDerive(
     _Out_ QUIC_PACKET_KEY **NewKey
     );
 
+#include "quic_datapath.h"
```

Comment:
Instead of including this here, just put a declaration for `CXPLAT_QEO_CONNECTION` at the top of the file.

Reply:
Thanks for good catch, forgot to remove from prototyping